### PR TITLE
Restore custom workflows

### PR DESCRIPTION
**Pull request by dstepanov**

### CODE REVIEW

File: .github/workflows/graalvm.yml [hunk 1]
```diff
@@ -14,51 +14,109 @@ on:
...
+      - uses: actions/cache@v2.1.7
         with:
           path: ~/.gradle/caches
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-      - name: Setup GraalVM CE
-        uses: graalvm/setup-graalvm@v1
+      - name: Set up JDK
+        uses: actions/setup-java@v2
```
Comment (alvarosanchez) [inline .github/workflows/graalvm.yml:61]: v3?

File: .github/workflows/graalvm.yml [hunk 2]
```diff
@@ -14,51 +14,109 @@ on:
...
-          then
-            ./gradlew check testNativeImage --continue --no-daemon
-          else
-            ./gradlew check --continue --no-daemon
-          fi
+          set -e
+          for EXAMPLE in $(ls ./examples); do
+            ./gradlew ${EXAMPLE}:test --continue --no-build-cache
+          done
+        env:
```
Comment (alvarosanchez) [inline .github/workflows/graalvm.yml:81]: You could keep `PREDICTIVE_TEST_SELECTION`

File: .github/workflows/gradle.yml [hunk 3]
```diff
@@ -14,59 +14,87 @@ on:
...
         env:
-           TESTCONTAINERS_RYUK_DISABLED: true
-           GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
-           GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GRADLE_ENTERPRISE_CACHE_USERNAME }}
-           GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GRADLE_ENTERPRISE_CACHE_PASSWORD }}
-           PREDICTIVE_TEST_SELECTION: "${{ github.event_name == 'pull_request' && 'true' || 'false' }}"
+          TESTCONTAINERS_RYUK_DISABLED: true
+          GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+          GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GRADLE_ENTERPRISE_CACHE_USERNAME }}
+          GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GRADLE_ENTERPRISE_CACHE_PASSWORD }}
```
Comment (alvarosanchez) [inline .github/workflows/gradle.yml:83]: Maybe keep `PREDICTIVE_TEST_SELECTION`

**Review by alvarosanchez — CHANGES_REQUESTED** [review]

**Comment by dstepanov** [comment]
Migrated some of the values from the shared workflows.

File: .github/workflows/graalvm.yml [hunk 4]
```diff
@@ -14,51 +14,111 @@ on:
...
       - uses: actions/cache@v3
         with:
           path: ~/.gradle/caches
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-      - name: Setup GraalVM CE
-        uses: graalvm/setup-graalvm@v1
+      - name: Set up JDK
+        uses: actions/setup-java@v2
```
Comment (alvarosanchez) [inline .github/workflows/graalvm.yml:61]:
```suggestion
        uses: actions/setup-java@v3
```

File: .github/workflows/gradle.yml [hunk 5]
```diff
@@ -36,12 +58,14 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Set up JDK
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v2
```
Comment (alvarosanchez) [inline .github/workflows/gradle.yml:61]:
```suggestion
        uses: actions/setup-java@v3
```

**Review by alvarosanchez — APPROVED** [review]

**Pull request merged**